common/src/main/java/com/wynntils/features/user/overlays/map/MiniMapOverlayFeature.java

```diff
@@ -136,6 +140,55 @@ private void rotateMapToPlayer(PoseStack poseStack, float centerX, float centerZ
             poseStack.translate(-centerX, -centerZ, 0);
         }
 
+        // TODO move most of the buffer builder code into RenderUtils, maybe separate each map border into different
```

Comment:
We can remove this todo by just making a method for this in RenderUtils? The other part of todo is already done